pillar/top.sls

```diff
@@ -40,6 +40,15 @@ base:
   'roles:fluentd':
     - match: grain
     - fluentd
+  'roles:zookeeper':
```

Comment:
Cloud profile file seems to be setting the role name as `zookeeper_server`. Same for bookkeeper. Guess either this file file needs to be changed or the cloud profile.s